distributor/src/distributor/filters.bal

```diff
@@ -95,7 +97,7 @@ public type SubscriptionFilter object {
 
         string headerValue = request.getHeader(http:AUTH_HEADER);
         
-        if !(headerValue.startsWith(auth:AUTH_SCHEME_BASIC)) {
+        if (!headerValue.startsWith(auth:AUTH_SCHEME_BASIC)) {
```

Comment:
Note - This is swapped here because otherwise no point of wrapping the expression with parentheses.